easybuild/easyconfigs/d/DFT-D3/DFT-D3-3.1.1-intel-2015a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'MakeCp'
+
+name = 'DFT-D3'
+version = '3.1.1'
+
+homepage = 'http://www.thch.uni-bonn.de/tc/index.php?section=downloads&subsection=DFT-D3&lang=english'
+description = """DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical quantum chemical methods."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://www.thch.uni-bonn.de/tc/downloads/%(name)s/data']
+sources = ['dftd3.tgz']
+checksums = [('md5', '88470bd398b5cc90ad680ed1df4239f7')]
```

Comment:
I'm now getting a different checksum, so was a new version released?

I'd suggest to use `dftd3-%(version)s.tgz` as source file name, implying that it should be downloaded manually and renamed (until we have support for automagically renaming on download in the framework), since this is not going to work out well otherwise, clearly...